java/src/jmri/jmrix/openlcb/OlcbLightManager.java

```diff
@@ -125,7 +123,8 @@ public boolean allowMultipleAdditions(String systemName) {
 
     @Override
     public boolean validSystemNameConfig(String address) throws IllegalArgumentException {
-        OlcbAddress a = new OlcbAddress(address);
+        String withoutPrefix = address.replace("ML", "");
```

Comment:
This feels hacky. If anyone has a better suggestion I'm all ears! After I wrote this, I noticed other classes take a substring that skips the length of the prefix, which is a better way.

Reply:
Take a look at how OlcbTurnoutManager does it.  That's a good model.